README.md

```diff
@@ -125,6 +125,10 @@ Namelist Option : `file_vars`  Full name of input file (Supports either text or
 - See example file inputs for variables and format (`gfs.t12z.20220701.sfcf000.canopy.txt` or `gfs.t12z.20220701.sfcf000.canopy.nc`).  Example surface met/land/soil inputs are based on NOAA's UFS-GFSv16 inputs initialized on July 01, 2022 @ 12 UTC (forecast at hour 000). Other external inputs for canopy related and other calculated variables are from numerous sources.  See [Table 2](#table-2-canopy-app-required-input-variables) below for more information.  **Note:** The example GFSv16 domain has been cut to the southeast U.S. region only in this example for size/time constraints here.
 - Canopy-App assumes the NetCDF input files are in CF-Convention and test file is based on UFS-GFSv16; recommend using double or float for real variables.  Input data must be valid values.
 - Canopy-App can also be run with a single point of 1D input data in a text file (e.g. `point_file_20220701.sfcf000.txt`).
+- The namelist can be modified with f90nml (included with the Canopy-App Conda environment) to insert multiple input filenames at once:
```

Comment:
```suggestion
- The namelist can be modified with f90nml (included with the [Canopy-App Conda](https://github.com/noaa-oar-arl/canopy-app/blob/develop/python/environment.yml) environment) to insert multiple input filenames at once:
```

Reply:
@MaggieMarvin  I added the inserted hyperlink to point to the environment file for further clarification.

Reply:
Relative path (`./python/environment.yml`) is another option for the link target.

Reply:
Also could link https://f90nml.readthedocs.io/en/latest/cli.html for f90nml as well?